docs/concepts/07-transferability.md

```diff
@@ -1,11 +1,11 @@
 ---
 id: "transferability"
-sidebar_position: 11
+sidebar_position: 7
 title: "Transferability"
```

Comment:
NFT section talks about the SVG representation of the streams where Transferability speaks about the transferability of the stream. Technically, both of them mean the same thing (since 1 stream = 1 NFT) but a user interested into checking the transferability of the stream might not be interested into its NFT representation and vice versa. That was my rationale into not merging them into one. LMK what you think.

Reply:
Agree with Shub.